integrations/FacebookPixel/browser.js

```diff
@@ -73,7 +74,7 @@ class FacebookPixel {
 
   page(rudderElement) {
     const { properties, messageId } = rudderElement.message;
-    window.fbq("track", "PageView", properties, { eventID: messageId });
+    window.fbq("track", "PageView", properties, { getEventId(message) });
```

Comment:
shouldn't this be `{ event_id: getEventId(message) }` ?
```suggestion
    window.fbq("track", "PageView", properties, { event_id: getEventId(message) });
```

Reply:
Done